docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,65 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+For experienced developers, here is a summary of the guide, see each section for more details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) follows instructions on their site for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Open a text editor and create a `.wp-env.json` file with the following content:
+
+```json
+{
+	"core": null
+}
+```
+
```

Comment:
Creating the json file:
![Create-wp-env-json-file](https://user-images.githubusercontent.com/5323259/91844429-bdde6500-ec57-11ea-99c4-063564a58580.png)

(It is not wise to use the regular text editor on the Mac as it automatically inserts an extension of .rtf which I later on be removed.) 

To show hidden files on the Mac: Press Command (cmd) + Shift + . (dot) to show or hide hidden files.

Reply:
Result:

```
paaljoachimromdahl@Paals-MacBook-Pro ~ % wp-env start
✖ Invalid .wp-env.json: Unexpected token \ in JSON at position 1
paaljoachimromdahl@Paals-MacBook-Pro ~ % 
```

Trying the troubleshooting instructions further below:

If you see the error: `No .wp-env.json file found` or `Invalid .wp-env.json`
- You need to create a valid .wp-env.json file. Try a file with just `{}`

I also quit and restarted terminal.
The error remains. I will use another editor to create the file. Looking at the recommended list, and I am downloading Atom.

Reply:
I will test and probably can remove the .wp-env.json steps.